src/hooks/useRGB.js

```diff
@@ -0,0 +1,13 @@
+import { useParams } from 'react-router-dom';
+
+export default function RGB() {
```

Comment:
So, this isn't a custom hook. This is actually a simple React Functional Component, so it should be in the components folder, along side Main and Header.